servers/partner-ci/common.yaml

```diff
@@ -86,35 +114,30 @@
       - string:
           name: PKG_JOB_BUILD_NUMBER
           description: 'The name of job for copying of plugin pkg artefacts'
-      - bool:
-          name: KVM_USE
-          default: '{obj:kvm_use}'
-          description: 'turns on kvm_mode for an openstack cluster'
-      - bool:
-          name: FORCE_DISABLE_UPDATES
-          default: False
-      - bool:
-          name: DRIVER_ENABLE_ACPI
-          default: False
-      - bool:
-          name: MAKE_SNAPSHOT
-          default: False
-      - bool:
-          name: FUEL_STATS_ENABLED
-          default: False
-      - bool:
-          name: ALWAYS_CREATE_DIAGNOSTIC_SNAPSHOT
-          default: False
-      - bool:
-          name: DEBUG_MODE
-          default: True
-      - bool:
-          name: GIT_CURL_VERBOSE
-          default: False
-      - bool:
-          name: DISCONNECT_ON_FAILURE
-          default: True
-
+      - string:
+          name: ENV_PREFIX
+          default: '{fuel-release}.{plugin-name}'
+          description: 'The name of environment'
+      - string:
+          name: PLUGIN_VERSION
+          default: '{plugin-version}'
+      - text:
+          name: COMMON_PROPS
+            default: |
+            SNAPSHOT_ARTIFACTS_FILE=snapshots.params
+            OPENSTACK_RELEASE=Ubuntu
+            NODES_COUNT=10
+            DESTROY_ENV_AFTER=0
+            ADMIN_NODE_MEMORY=4096
```

Comment:
need to redefine to null variables for changing amount of a node resources for projects, where DEVOPS_TEMPLATE is used